src/utils/sqsProcess.ts

```diff
@@ -0,0 +1,14 @@
+import DynamoDB from "aws-sdk/clients/dynamodb";
+
+export const processRecord = (record: any) => {
+    const recordBody = JSON.parse(JSON.parse(record.body).Message);
+    if (
+      recordBody.eventName === "MODIFY"
+      && recordBody.dynamodb
+      && recordBody.dynamodb.NewImage
+    ) {
+      return DynamoDB.Converter.unmarshall(recordBody.dynamodb.NewImage);
+    }
+    console.error(`process return undefined, record is ${JSON.stringify(recordBody)}`);
```

Comment:
Agree with some sort of log being made here, as atf-report-gen should only generate the report once the visit has ended. Just wondering if it should be a WARN or INFO rather than error, as it's more of a 'no action required' - what are your thoughts?

Reply:
Yeah will change to a warning I think, thanks